### DISCUSSION TITLE
How to install certain version of flux cli

### DISCUSSION THREAD

**Discussion by crmagz** [fluxcd/flux2#2396] [Q&A] [answered]
Trying to install 0.24.0 by setting env FLUX_VERSION=0.24.0 and running `curl -s https://fluxcd.io/install.sh | sudo bash` but still getting the latest. Any suggestions on how to pull a certain version of flux cli?

**Comment by tjanson** [comment, marked as answer]
Are you sure the env var is set inside the bash shell you're opening with sudo? Edit: use `sudo -E` to make sure the env var is passed along. (See `--preserve-env` in `man sudo` for details.)
